src/Gallery.js

```diff
@@ -1,11 +1,6 @@
 import React from 'react';
-import niceGuysFc from './pics/niceguys.jpg';
-import suitsFc from './pics/suits.jpg';
-import narutoFc from './pics/naruto.jpg';
-import howimetyourmotherFc from './pics/howimetyourmother.jpg';
-import breakingbadFc from './pics/breakingbad.jpg';
-import alteredcarbonFc from './pics/alteredcarbon.jpg';
 import { Link } from "react-router-dom";
+import films from './filmObject';
```

Comment:
I find `filmObject` a bit confusing, as it returns an array. Can you rename it to e.g. `filmsList`?